instaloader/structures.py

```diff
@@ -1061,6 +1061,20 @@ def get_followers(self) -> NodeIterator['Profile']:
             'https://www.instagram.com/{0}/'.format(self.username),
         )
 
+    def get_followed_hashtags(self) -> Iterator['Hashtag']:
+        """
+        Retrive list of hashtags followed by given profile.
+        To use this, one needs to be logged in and private profiles has to be followed.
+        """
```

Comment:
We should also base this change on `upcoming/v4.10`, and add a `.. versionadded:: 4.10` directive in this docstring.